unison-src/transcripts/todo.output.md

```diff
@@ -130,4 +130,241 @@ structural type MyType = MyType Int
        introduce the same name. Use `move.term` or `delete.term`
        to resolve the conflicts.
 
+```
+## A named value that appears on the LHS of a patch isn't shown
+
+```unison
+foo = 801
+```
+
+```ucm
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These new definitions are ok to `add`:
+    
+      foo : Nat
+
+```
+```ucm
+.lhs> add
+
+  ⍟ I've added these definitions:
+  
+    foo : Nat
+
+```
+```unison
+foo = 802
+```
+
+```ucm
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These names already exist. You can `update` them to your
+      new definition:
+    
+      foo : Nat
+
+```
+```ucm
+.lhs> update
+
+  ⍟ I've updated these names to your new definition:
+  
+    foo : Nat
+
+```
+```unison
+oldfoo = 801
+```
+
+```ucm
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These new definitions are ok to `add`:
+    
+      oldfoo : Nat
+
+```
+```ucm
+.lhs> add
+
+  ⍟ I've added these definitions:
+  
+    oldfoo : Nat
+
+.lhs> view.patch patch
+
+  Edited Terms: 1. oldfoo -> 2. foo
+  
+  Tip: To remove entries from a patch, use
+       delete.term-replacement or delete.type-replacement, as
+       appropriate.
+
+.lhs> todo
+
+  ✅
+  
+  No conflicts or edits in progress.
+
+```
+## A type-preserving update to one element of a cycle, which doesn't (yet) propagate to the other
+
+```unison
+even = cases
+  0 -> true
+  n -> odd (drop 1 n)
+
+odd = cases
+  0 -> false
+  n -> even (drop 1 n)
+```
+
+```ucm
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These new definitions are ok to `add`:
+    
+      even : Nat -> Boolean
+      odd  : Nat -> Boolean
+
+```
+```ucm
+.cycle1> add
+
+  ⍟ I've added these definitions:
+  
+    even : Nat -> Boolean
+    odd  : Nat -> Boolean
+
+```
+```unison
+even = cases
+  0 -> true
+  2 -> true
+  n -> odd (drop 1 n)
+```
+
+```ucm
+
+  I found and typechecked these definitions in scratch.u. If you
+  do an `add` or `update`, here's how your codebase would
+  change:
+  
+    ⍟ These names already exist. You can `update` them to your
+      new definition:
+    
+      even : Nat -> Boolean
+
+```
+```ucm
+.cycle1> update
+
+  ⍟ I've updated these names to your new definition:
+  
+    even : Nat -> Boolean
+
+```
+```ucm
+.cycle1> todo
+
+  🚧
+  
+  The namespace has 2 transitive dependent(s) left to upgrade.
+  Your edit frontier is the dependents of these definitions:
```

Comment:
Unrelated issue, but noting that "edit frontier" isn't super clear without maybe a lot of docs and illustrations around the process.

Reply:
Very unclear, and even moreso because the code calls something *else* the "frontier".